### ISSUE TITLE
Array Resizing

### ISSUE THREAD

**Issue by CJ-Tran** [csci-1301/feedback-SU2021#15]
In the **Changing the Size** under **Array**, it might be helpful to also explain how to resize an array _without_ using the Resize method (i.e. creating a temporary array, iterating over each value with a loop, copying it over, setting the old array equal to the temporary array), since not using the Resize method could be helpful in reinforcing previous concepts (arrays and loops).

**Comment by etremel** [comment]
Yes, I actually included an example of how to do that when introducing the Resize method in my version of the course. I can add it to the Arrays section (which I haven't really looked at yet) sometime soon.
